ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh

```diff
@@ -293,14 +294,14 @@ fi
 
 pushd $CNF_REPO_DIR
 if [[ "$T5CI_VERSION" == "4.15" ]] || [[ "$T5CI_VERSION" == "4.16" ]]; then
+    check_for_pr "openshift-kni" "cnf-features-deploy"
```

Comment:
I think this needs to be outside of the if statement and not only for 4.15 and 4.16

Reply:
fixed thanks, we do want it to run on any other versions